src/views/Timesheets/components/ShiftItem/index.js

```diff
@@ -292,7 +307,7 @@ export default class ShiftItem extends Component {
     } else {
       timeInDisp = <Typography variant="body2">{dateTimeIn}</Typography>
       timeOutDisp = <Typography variant="body2">{dateTimeOut}</Typography>
-      hoursWorkedDisp = <Typography variant="body2">{HOURS_WORKED}</Typography>
+      hoursWorkedDisp = <Typography variant="body2">{HOURS_WORKED.toFixed(2)}</Typography>
```

Comment:
Did we set up the .toFixed(2) in every spot we should?

Reply:
oops